xrpl/models/requests/tx.py

```diff
@@ -17,18 +17,28 @@
 class Tx(Request):
     """
     The tx method retrieves information on a single transaction.
+    The Request must contain either transaction or CTID parameter, but not both.
 
     `See tx <https://xrpl.org/tx.html>`_
     """
 
     method: RequestMethod = field(default=RequestMethod.TX, init=False)
-    transaction: str = REQUIRED  # type: ignore
-    """
-    This field is required.
-
-    :meta hide-value:
-    """
-
+    transaction: Optional[str] = None
     binary: bool = False
     min_ledger: Optional[int] = None
     max_ledger: Optional[int] = None
+    ctid: Optional[str] = None
```

Comment:
While you're here, I'd recommend putting in the docstrings from xrpl.org as in-line comments like the old """ This field is required. """ - https://xrpl.org/tx.html
(Note that docstrings in python with our docs generation tool `Sphinx` happen AFTER the line you're trying to document as far as I understand)

I'd also include in those docstrings for `transaction` and `ctid` that either it or the other is required. 

Lastly, I'd re-order the parameters here so they match the order on xrpl.org, but that's a nit :)

Reply:
okay, updated in [ce890c0](https://github.com/XRPLF/xrpl-py/pull/672/commits/ce890c04ceb5fc35ef832fc1c8439e712c4cd3e0)